subcommand/server-acl-init/command_test.go

```diff
@@ -88,6 +88,7 @@ func TestRun_Defaults(t *testing.T) {
 		clientset: k8s,
 	}
 	args := []string{
+		"-timeout=1m",
```

Comment:
adding this timeout to all tests, so that if the test fails, it doesn't run too long.